Applications/TextEditor/TextEditorWidget.h

```diff
@@ -15,10 +16,12 @@ class TextEditorWidget final : public GWidget {
     void open_sesame(const String& path);
 
 private:
-    void set_path(const StringView&);
+    void set_path(FileSystemPath file);
```

Comment:
We should pass non-primitives by const-reference to avoid unnecessary copies.
```suggestion
    void set_path(const FileSystemPath&);
```